src/pages/plugins/getting-started/index.md

```diff
@@ -0,0 +1,115 @@
+  <figcaption>The create plugin dialog</figcaption>
+</figure>
+
+In the dialog, enter the following information:
+
+| Field                    | Value             |
+|--------------------------|-------------------|
+| Name                     | My First Plugin   |
+| Host Application         | Adobe InDesign    |
+| Host Application Version | 16.0              |
+| Template                 | starter           |
+
+[//]: # (TODO: Fix according to actual UDT data and version requirements)
+[//]: # (TODO: Add template "starter" with the code below to UDT)
+
+Then click **Select Folder** and choose a location on your machine where you want to create the plugin.
+
+The UXP Developer Tool will create a new folder with the name you specified in the location you specified. The folder will contain the following files:
+
+```
+My First Plugin
+├── manifest.json
+├── index.html
+└── index.js
+```
+
+### Step 2: Run the plugin
+
+Now that you have created your plugin, you can run it in InDesign.
+
+First things first, you'll need to run InDesign.
+
+Then, in the UXP Developer Tool, in the row for your plugin, click **⋯ &rarr; Load**:
+
+<figure>
+  <img src="https://picsum.photos/1280/720" alt="Screenshot of the UXP Developer Tool with the load button highlighted" />
+  <figcaption>The load button</figcaption>
+</figure>
+
+This will load your plugin into InDesign. You should see the plugin's panel appear in InDesign:
+
+<figure>
+  <img src="https://picsum.photos/1280/720" alt="Screenshot of InDesign with the plugin panel open" />
+  <figcaption>The plugin panel</figcaption>
+</figure>
+
+### Step 3: Make a change
+
+Now that you have your plugin running in InDesign, let's make this plugin your own.
+
+Open the `index.html` file in your code editor. You should see the following code:
+
+```html
+<html>
+  <head>
+    <script src="./index.js"></script>
+  </head>
+  <body>
+    <h1>Hello World!</h1>
+    <button id="btnCreateText">Create Text</button>
+  </body>
+</html>
+```
+
+This is the HTML code that defines the content of your plugin's panel. You can change it to whatever you want.
+
+For example, you could change the heading to say "My First Plugin":
+
+```html
+<h1>My First Plugin</h1>
+```
+
+Or you could add a paragraph below the heading:
+
+```html
+<p>This is my first plugin.</p>
+```
+
+To see your changes, you'll need to reload the plugin in InDesign. To do so, in the UXP Developer Tool, in the row for your plugin, click **⋯ &rarr; Reload**:
+
+<figure>
+  <img src="https://picsum.photos/1280/720" alt="Screenshot of the UXP Developer Tool with the reload button highlighted" />
+  <figcaption>The reload button</figcaption>
+</figure>
+
+<InlineAlert slots="text" />
+
+If you make any changes to the `manifest.json` file, you'll need to fully unload and reload the plugin. To do so, in the UXP Developer Tool, in the row for your plugin, click **⋯ &rarr; Load**:
```

Comment:
Replace click **⋯ &rarr; Load**: with
click **⋯ &rarr; Unload, Reload**:

Reply:
In my experience, it's possible with just "Load" (as this internally unloads and loads it again). The only thing that doesn't work is "Reload". So this should be possible as it is, but if it makes sense for didactic reasons, we can of course change it.

Reply:
About the bolded arrows, there is a style guide for writing tutorials, please try to adhere to the guidelines as closely as possible: https://zeroheight.com/58b352410/p/053578-writing-for-tutorialsprocedural-pieces/b/44c264